react/features/subtitles/middleware.ts

```diff
@@ -11,7 +12,7 @@ import {
 import {
     removeTranscriptMessage,
     updateTranscriptMessage
-} from './actions';
+} from './actions.web';
```

Comment:
If this middleware does not run on mobile, please rename it to middleware.web or this is bound to eventually break.

Reply:
It's currently imported on middlewares.any ...

Reply:
Fixed it